packages/web3-utils/src/chunk_response_parser.ts

```diff
@@ -63,14 +70,17 @@ export class ChunkResponseParser {
 				}
 
 				this.lastChunkTimeout = setTimeout(() => {
+					if (this.autoReconnect) return;
 					this.clearQueues();
-					throw new InvalidResponseError({
-						id: 1,
-						jsonrpc: '2.0',
-						error: { code: 2, message: 'Chunk timeout' },
-					});
+					this.eventEmitter.emit(
```

Comment:
ChunkResponseParser should not be timing out the program, instead emit an error to websocket that chunk has timed out.